lib/config.py

```diff
@@ -100,6 +100,25 @@
     ('--packages-dir', '-d'):
         dict(help='Directory of packages used in the ISO image.',
              default='./result'),
+    ('--iso-name',):
+        dict(help='ISO name.',
+             default='OpenPOWER-Host_OS'),
```

Comment:
I prefer to remove all the default values from the command line parser in a separate patch